mscore/transposedialog.cpp

```diff
@@ -48,39 +48,47 @@ TransposeDialog::TransposeDialog(QWidget* parent)
       setupUi(this);
       setWindowFlags(this->windowFlags() & ~Qt::WindowContextHelpButtonHint);
 
-      connect(transposeByKey, SIGNAL(clicked(bool)), SLOT(transposeByKeyToggled(bool)));
-      connect(transposeByInterval, SIGNAL(clicked(bool)), SLOT(transposeByIntervalToggled(bool)));
+      connect(transposeByKey,       SIGNAL(clicked(bool)),  SLOT(transposeByKeyToggled(bool)));
+      connect(transposeByInterval,  SIGNAL(clicked(bool)),  SLOT(transposeByIntervalToggled(bool)));
+      connect(chromaticBox,         SIGNAL(clicked(bool)),  SLOT(chromaticBoxToggled(bool)));
+      connect(diatonicBox,          SIGNAL(clicked(bool)),  SLOT(diatonicBoxToggled(bool)));
```

Comment:
While we are at it, maybe we should move to the [new signal-slot connection syntax](https://wiki.qt.io/New_Signal_Slot_Syntax) here? This would make all connections be checked at compile time so we won't miss possible error because of any slot name/signature changes.

Reply:
There're a LOT of instances of this kind of old syntax in the codebase right now. Maybe this switch should belong to a different PR which deals with all of them?

Reply:
I believe that it's **because** there is so many that one pull request can change them all.
It's the same with Qt's foreach. It'd be really easy to remove them all at once, but it would cause so many merge conflicts, that it would be a pain more anything else.
From my understanding, the goal is to gradually update the code base, to avoid these merge conflicts.
You might find some more arguments in [this old PR](https://github.com/musescore/MuseScore/pull/3879)

Reply:
My view too

Reply:
And besides the update in the developers' handbook, it can be part of the review too.
So, for this PR, dmitrio95's comment sound as a good idea to me.

Reply:
Yes, I agree, it probably makes more sense to replace this gradually.

Reply:
Done.